### ISSUE TITLE
Using loss function without trainers

### ISSUE THREAD

**Issue by alisonbma** [KevinMusgrave/pytorch-metric-learning#393]
Hi! I just wanted to clarify if the library supports the use of loss functions and miners without the trainer? I'd like to use them with my own configured training loop so that I can better control the forward/backward calls, etc.

**Comment by KevinMusgrave** [comment]
Yes, see [this part of the readme](https://github.com/KevinMusgrave/pytorch-metric-learning#using-losses-and-miners-in-your-training-loop)

**Comment by alisonbma** [comment]
> Yes, see [this part of the readme](https://github.com/KevinMusgrave/pytorch-metric-learning#how-loss-functions-work)

Got it, thanks!
